app/controllers/main/routes.py

```diff
@@ -156,10 +160,103 @@ def viewUsersProfile(username):
                 currentDateTime = datetime.datetime.now(),
                 profileNotes = profileNotes,
                 bonnerRequirements = bonnerRequirements,
-                userDiet = userDiet                
+                userDiet = userDiet
             )
     abort(403)
 
+@main_bp.route('/profile/<username>/emergencyContact', methods=['GET', 'POST'])
+def emergencyContactInfo(username):
+    """
+    This loads the Emergency Contact Page
+    """
+    if not (g.current_user.username == username or g.current_user.isCeltsAdmin):
+        abort(403)
+
+
+    if request.method == 'GET':
+
+        # Beans: Q, Should student staff be able to see the emergency contact and insurance info of everyone?
+        readOnly = False if g.current_user.username == username else True
```

Comment:
u can just do `readOnly = g.current_user.username != username`

it evaluates the way you want it to. the only problem is readability which tbh is a you problem